package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "jss-styled",
+  "version": "0.0.1",
+  "description": "Styled primitives with JSS",
+  "main": "lib/index.js",
+  "scripts": {
+    "clean": "rm -rf lib/*",
+    "lint": "eslint src",
+    "test": "jest",
+    "test:watch": "npm test -- --watch",
+    "test:coverage": "npm test -- --coverage",
+    "prebuild": "npm run clean && npm run lint && npm run test:coverage",
+    "build": "babel src --out-dir lib --ignore tests",
+    "postinstall": "flow-typed install",
+    "flow:stub": "flow-typed create-stub"
+  },
+  "files": [
+    "src",
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lttb/jss-styled.git"
+  },
+  "keywords": [
+    "jss",
+    "styled"
+  ],
+  "author": "lttb <kenzhaev.artur@gmail.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/lttb/jss-styled/issues"
+  },
+  "homepage": "https://github.com/lttb/jss-styled#readme",
+  "dependencies": {
+    "jss": "^6.5.0",
+    "jss-preset-default": "^1.3.1",
+    "react": "^15.4.2",
+    "react-jss": "^5.4.0",
+    "recompose": "^0.23.1"
+  },
+  "devDependencies": {
+    "@lttb/eslint-config-default": "github:lttb/configs#js",
```

Comment:
in case we are going to move this repo to the cssinjs org, we should use https://github.com/cssinjs/eslint-config-jss